components/Manual.tsx

```diff
@@ -511,14 +511,32 @@ function Manual(): React.ReactElement {
                   />
                   <div className="mt-4 pt-4 border-t border-gray-200">
                     {pageList[pageIndex - 1] !== undefined && (
-                      <Link href={pageList[pageIndex - 1].path}>
+                      <Link
+                        href={
+                          version
+                            ? pageList[pageIndex - 1].path.replace(
+                                "manual",
+                                `manual@${version}`
+                              )
+                            : pageList[pageIndex - 1].path
+                        }
+                      >
                         <a className="text-gray-900 hover:text-gray-600 font-normal">
                           ← {pageList[pageIndex - 1].name}
                         </a>
                       </Link>
                     )}
                     {pageList[pageIndex + 1] !== undefined && (
-                      <Link href={pageList[pageIndex + 1].path}>
+                      <Link
+                        href={
+                          version
```

Comment:
It seems that `version` can never be null/undefined here. So the next of `/manual/introduction` is no longer `/manual/gettings_started`, but `/manual@LATEST/gettings_started` now. But I think that's fine because we already uses versioned urls in the side menu. So this is consistent with how the side menu navigates.